tornettools/generate_tgen.py

```diff
@@ -368,8 +423,12 @@ def __get_tgen_clients(args, n_users, n_circuits, measurement2):
     # can already be adjusted with args.process_scale.
     n_circuits_per_tgen = round(n_circuits / n_tgen * args.load_scale)
 
+    # split the tgen clients between onion-service clients and non-onion-service (public) clients
+    n_hs_tgen = round(n_tgen * args.onion_service_traffic_scale)
```

Comment:
`ceil` instead of `round` so we have at least one if `args.onion_service_traffic_scale` is positive?

Reply:
Added.